Sources_v3/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -69,6 +90,16 @@ open class ChatChannelListVC<ExtraData: UIExtraDataTypes>: UIViewController,
         guard bottomEdge >= scrollView.contentSize.height else { return }
         controller.loadNextChannels()
     }
+    
+    // MARK: Actions
+    
+    @objc private func didTapOnCurrentUserAvatar(_ sender: UITapGestureRecognizer) {
+        didSelectCurrentUser()
+    }
+    
+    @objc private func didTapCreateNewChannel(_ sender: UIBarButtonItem) {
+        didSelectCreateNewChannel()
+    }
```

Comment:
Instead of `private` these methods should rather be `public/open` :)

Reply:
Hm.. We use closures to handle the actions -> end-users can provide any handlers. Do you think we want those methods to be `public/open`?

Reply:
I'd say so. Let's try to use the public-by-default approach. Unless there's a serious reason why not to make it private, let's make it public.

Reply:
made it `open`